src/RadhydroSimulation.hpp

```diff
@@ -44,14 +44,22 @@
 #include "AMReX_YAFluxRegister.H"
 
 #include "Chemistry.hpp"
+
+#ifndef CHEM
 #include "CloudyCooling.hpp"
+#endif
+
```

Comment:
wrapped up CloudyCooling in #ifndef CHEM: now, we either compile CloudyCooling OR Chemistry, but not both.